content/en/database_monitoring/guide/connect_dbm_and_apm.md

```diff
@@ -144,6 +145,56 @@ client.query("SELECT 1;")
 
 {{% /tab %}}
 
+{{% tab "NodeJS" %}}
+
+Install or udpate [dd-trace-js][1] to version greater than `3.9.0` (or `2.22.0` if using end-of-life Node.js version 12):
+
+```
+npm install dd-trace@^3.9.0
+```
+
+Update your code to import and initialize the tracer:
+```javascript
+// This line must come before importing any instrumented module.
+const tracer = require('dd-trace').init();
+```
+
+Enable the database monitoring propagation feature using one of the following methods:
+1. Env variable:
+   `DD_DBM_PROPAGATION_MODE=full`
+
+2. Option `dbmPropagationMode` (default: `ENV['DD_DBM_PROPAGATION_MODE']`):
+   ```javascript
+	tracer.use('pg', { dbmPropagationMode: 'full', service: 'my-db-service' })
+   ```
+
+Full example:
+```javascript
+const tracer = require('dd-trace').init()
+
+tracer.use('pg', { dbmPropagationMode: 'full', service: 'my-db-service' })
+
+const client = new pg.Client({
+	user: 'postgres',
+	password: 'postgres',
+	database: 'postgres'
+})
+
+client.connect(err => done(err))
```

Comment:
Maybe change this to:

```js
client.connect(err => {
  console.error(err);
  process.exit(1);
});
```

As the `done` function seems like a test suite convention